src/utils/isCustomPropertySet.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Check whether a property is a custom properties
+ *
+ * @param {string} customPropertySet
+ * @return {boolean} If `true`, property is a custom properties
+ */
+export default function (customPropertySet) {
+  return (customPropertySet.slice(0, 2) === "--" && customPropertySet.slice(-1) === ":")
+}
```

Comment:
This is almost exactly the same as https://github.com/stylelint/stylelint/blob/master/src/utils/isCustomProperty.js, which is kind of confusing. And it does not really distinguish between a custom property set and a regular custom property, right?